### ISSUE TITLE
Solid notifications: take into account unfetchable activities

### ISSUE THREAD

**Issue by srosset81** [assemblee-virtuelle/activitypods#194]
Mastodon sends activities which cannot be dereferenced, for example for `Accept` activities. They do not appear on the outbox of the user, even though they are sent to recipients.

We [adapted SemApps code to handle such case](https://github.com/assemblee-virtuelle/semapps/pull/1161/commits/eff99d4dd2f543b70c6775866dda657e84faf195) to handle such case, but we need to adapt Solid notifications code so that the full activity is sent in this case. Otherwise the external applications will not be able to know what this activity was about.

Basically when the `activitypub.collection.added` is sent with a `item` (and not a `itemUri`), we should pass the full object.

**Comment by srosset81** [comment]
Done here https://github.com/activitypods/activitypods/commit/661b888dbc77a25282ac94450ac302fd5572d2f6
